### ISSUE TITLE
Agency and the AlignmentObject

### ISSUE THREAD

**Issue by stuartasutton** [CredentialTransparencyInitiative/domain-models#2]
[From Robby Robson] The AlignmentObject and the mechanisms in the draft credential data model have no agency. This is problematic because it can be important to know not only what is asserted about a competency or credential, but also who asserted it and whether the assertion is still valid. Agency is part of the Mozilla Open Badge standard and is included in the assertions that an Organization can make, but there may be simpler approaches.
 
For example, agency can be handled with some limitations if the AlignmentObject is a property of the credential itself and there is a default agent. (The same applies to competencies.) Say, for example, that the creator is assumed to be the creator and asserter of all relationships within the credential. This defines the agent in most cases, and a different agent can create or assert a different relationship between objects A and B by creating a duplicate object A’, declaring it to be the same as A, and then declaring the relationship within A’.

**Comment by stuartasutton** [comment]
Bingo! AlignmentObject in schema.org has served as a (kind of) reification of the property identifying the relationship between the CreativeWork being described and the framework being referenced making it possible to say things about the relationship (i.e., identify kind of alignment/relationship, framework being referenced etc.). As you note, Robby, one could very easily see agency --i.e., who makes this assertion of alignment-- as one more aspect of the AlignmentObject description here in the credential context (or even a learning resource context).

**Comment by jgoodell2** [comment]
Yes! I see value in adding agency (who makes this assertion of alignment) to AlignmentObject.
